packages/test/stochastic-test-utils/src/describeFuzz.ts

```diff
@@ -95,11 +121,22 @@ export function createFuzzDescribe(optionsArg?: FuzzDescribeOptions): DescribeFu
 			? Number.parseInt(process.env.FUZZ_TEST_COUNT, 10)
 			: undefined;
 	const testCount = testCountFromEnv ?? options.defaultTestCount;
-	const isStress =
-		process.env?.FUZZ_STRESS_RUN !== undefined && !!process.env?.FUZZ_STRESS_RUN;
-	const args = { testCount, isStress };
+
+	const stressMode = (() => {
+		switch (process.env?.FUZZ_STRESS_RUN) {
+			case "normal":
+				return StressMode.Normal;
+			case "long":
+				return StressMode.Long;
+			case "short":
+			default:
+				return StressMode.Normal;
+		}
+	})();
+
+	const args = { testCount, stressMode };
 	const d: DescribeFuzz = (name, tests) =>
-		(isStress ? describe.only : describe)(name, createSuite(tests, args));
+		(stressMode != null ? describe.only : describe)(name, createSuite(tests, args));
```

Comment:
```suggestion
		(stressMode != StressMode.Short ? describe.only : describe)(name, createSuite(tests, args));
```

otherwise we will end up skipping tests.